packages/augur-tools/docker-builder/run.sh

```diff
@@ -4,6 +4,6 @@ IMAGE=$1
 TAG=$(node ./scripts/get-contract-hashes.js)
 ./scripts/copy-docker-files.sh $IMAGE:$TAG
 echo "Running Geth with contracts deployed from ${IMAGE}:${TAG}"
-docker pull $IMAGE:$TAG
+#docker pull $IMAGE:$TAG
```

Comment:
Is this debug code?

Reply:
no, we don't need that, running the image will pull from docker hub, but if you have it locally the pull will overwrite it. We should just delete the whole line

Reply:
We actually just decided NOT to do that.

@petong lets just remove?

```suggestion
```

Reply:
```suggestion
```

Shall we remove it @petong